Automattic/wp-calypso
# Gravatar: Adjust the tracking events & add a why WP.com link

Related to p9lV3a-6su-p2

## Proposed Changes

* Make Gravatar powered magic login relevant events more generic
* Add why WP.com account doc link

## Testing Instructions



* Check out this PR
* Log out your account
* Go to the magic login of [Gravatar](http://calypso.localhost:3000/log-in/link?client_id=1854&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fclient_id%3D1854%26response_type%3Dcode%26blog_id%3D0%26state%3Dbfa7ea95b0a07d39e3f23e1d923ce77b35016295d986b51361d4334a6e571206%26redirect_uri%3Dhttps%253A%252F%252Fen.gravatar.com%252Fconnect%252F%253Faction%253Drequest_access_token%26from-calypso%3D1), and you will see the new "a WordPress.com account" link in the ToS section
* Go to the magic login of [Gravatar](http://calypso.localhost:3000/log-in/link?client_id=1854&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fclient_id%3D1854%26response_type%3Dcode%26blog_id%3D0%26state%3Dbfa7ea95b0a07d39e3f23e1d923ce77b35016295d986b51361d4334a6e571206%26redirect_uri%3Dhttps%253A%252F%252Fen.gravatar.com%252Fconnect%252F%253Faction%253Drequest_access_token%26from-calypso%3D1) and [WPJM](http://calypso.localhost:3000/log-in/link/?client_id=90057&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthenticate%2F%3Fclient_id%3D90057%26response_type%3Dcode%26state%3D01b7d7bfcfd126450053529ce09fefb5%26redirect_uri%3Dhttp%253A%252F%252Fwpjobmanagercom.vipdev.lndo.site%253A8000%252Fwp-json%252Fwpjmcom-auth%252Fv1%252Fcallback&create_account=1), and check these events have been sent correctly:

`calypso_gravatar_powered_magic_login_email_form`
<img width="1610" alt="截圖 2023-09-19 下午4 12 36" src="https://github.com/Automattic/wp-calypso/assets/21308003/d00d82f4-18b2-4659-8d41-340b2b9035fc">

`calypso_gravatar_powered_magic_login_click_login_page_link`
<img width="1627" alt="截圖 2023-09-19 下午4 13 15" src="https://github.com/Automattic/wp-calypso/assets/21308003/a0025fc0-aa4f-4c7a-9409-0f8d7d4ccb6b">

`calypso_gravatar_powered_magic_login_email_verification`
<img width="1597" alt="截圖 2023-09-19 下午4 14 41" src="https://github.com/Automattic/wp-calypso/assets/21308003/d99c243a-4222-4095-b7c9-4eb2421f3d8e">

`calypso_gravatar_powered_magic_login_click_use_different_email`
<img width="1608" alt="截圖 2023-09-19 下午4 15 22" src="https://github.com/Automattic/wp-calypso/assets/21308003/b919c75f-bbbc-4433-acd2-d3b07d26df67">

`calypso_gravatar_powered_magic_login_click_resend_email`
<img width="1607" alt="截圖 2023-09-19 下午4 18 27" src="https://github.com/Automattic/wp-calypso/assets/21308003/d93b105b-8fde-4b08-b9d6-f8c9a3c88295">

## Pre-merge Checklist
